fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -475,6 +477,32 @@ public static void dropStatisticsAfterDropTable(Table table) {
         GlobalStateMgr.getCurrentState().getStatisticStorage().expireConnectorTableColumnStatistics(table, columns);
     }
 
+    /**
+     * Change the replication_num of system table according to cluster status
+     * 1. When scale-out to greater than 3 nodes, change the replication_num to 3
+     * 3. When scale-in to less than 3 node, change it to retainedBackendNum
+     */
+    public static boolean alterSystemTableReplicationNumIfNecessary(String tableName) {
```

Comment:
```
Behavior change:

before: the replication_num will only be increased but not decreased
now: the replication_num will be increased when scale-out, and decreased when scale-in
```

plz add test case for this behavior change

Reply:
done